src/wasm/source-map.cpp

```diff
@@ -33,96 +34,55 @@ void MapParseException::dump(std::ostream& o) const {
   Colors::normal(o);
 }
 
-void SourceMapReader::readHeader(Module& wasm) {
-  assert(pos == 0);
+void SourceMapReader::parse(Module& wasm) {
   if (buffer.empty()) {
     return;
   }
-
-  auto skipWhitespace = [&]() {
-    while (pos < buffer.size() && (buffer[pos] == ' ' || buffer[pos] == '\n')) {
-      ++pos;
+  json::Value sm_json;
```

Comment:
```suggestion
  json::Value json;
```
We don't use camel case generally, and `json` seems clear enough?